sec-scanners-config.yaml

```
@@ -0,0 +1,8 @@
+module-name: keda-manager
+protecode:
+  - eu.gcr.io/kyma-project/keda-manager:v20230405-03a2ca00
```

```suggestion
  - eu.gcr.io/kyma-project/keda-manager:v20230405-03a2ca00
  - ghcr.io/kedacore/keda:2.10.0
  - ghcr.io/kedacore/keda-admission-webhooks:2.10.0
  - ghcr.io/kedacore/keda-metrics-apiserver:2.10.0
```

We should scan keda itslef as well